core/contracts/Voting.sol

```diff
@@ -263,6 +271,13 @@ contract Voting is Testable, MultiRole, OracleInterface, VotingInterface, Encryp
         }
     }
 
+    /**
+     * @notice Disables this Voting contract in favor of the migrated one.
+     */
+    function setMigrated(address newVotingAddress) external onlyRoleHolder(uint(Roles.Writer)) {
```

Comment:
It doesn't seem like this touches any of the voting mechanics. Does that mean that people will still be able to vote on existing requests without being rewarded (or penalized)?

Reply:
Yeah, in this implementation, `commit` and `reveal` still work, but voters have no incentive to ever call them. It's a simple check to turn those methods off, do you prefer doing that?

Reply:
As discussed offline, it's simpler to reason about if we disable voters from committing and revealing. I went back and forth on whether we should allow new voting to call those methods, currently leaning toward allowing but wdyt?

Reply:
My intuition is that it can't hurt to allow it. There is a potential upgrade path where new voting might want to allow old votes to be finalized in the old contract, so it may wish to forward on commits and reveals.